CONTRIBUTING.md

```diff
@@ -89,7 +89,10 @@ This will download the entire FCC repo to your projects directory.
 
 #### Setup Your Upstream
 
-1. Change directory to the new freeCodeCamp directory (`cd freeCodeCamp`)
+1. Change directory to the new freeCodeCamp directory  
+```bash
```

Comment:
This should be `shell` instead same as in rest of the guide.

Reply:
hey, `bash` is correct and take effect!